guides/source/active_record_querying.md

```diff
@@ -2187,6 +2187,74 @@ NOTE: Note that if a query matches multiple records, `find_by` will
 fetch only the first one and ignore the others (see the `LIMIT 1`
 statement above).
 
+### Using Common Table Expression (CTE)
```

Comment:
Currently this adds the section under "Understanding Method Chaining", which isn't related.
It probably deserves it own section, maybe after "Joining Tables"?
```suggestion
## Using Common Table Expressions (CTE)
```

Reply:
Moved CTE section to after "Joining Tables" section